### PR TITLE
LPS-168979 Change label for edit segment toggle

### PR DESCRIPTION
**Pull request by claraizquierdo**
Motivation
=======
The edit segment toggle label shouldn't change when it switches between active and inactive. See the _How to fix, recommendation_ section in the ticket description.
[Link to story or ticket](https://issues.liferay.com/browse/LPS-168979)

Proposed Solution
========
We have added a new text "Edit segment" as aria-label and title for the toggle.

Steps to Verify:
----------------
1. Navigate to People > Segments
2. Click the "+" to add a new segment
3. Active the screen reader you use
4. Traverse the add new segment page and focus on the Edit segment toggle
5. Observe SR announcement
6. Enable and disable the toggle and check that the label doesn't change. It should always be "Edit Segment".

### CODE REVIEW

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/segment_edit/SegmentEdit.es.js [hunk 1]
```diff
@@ -551,31 +551,19 @@ class SegmentEdit extends Component {
 										<div className="btn-group">
 											<div className="btn-group-item mr-2">
 												<ClayToggle
```
Comment (Stephy6) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/segment_edit/SegmentEdit.es.js:553]: I think the aria-label is not necessary, I've removed it and the label 'Edit Segment' still pop-up when I hover over or focus on the toggle. Maybe with the title is enough

**Review by Stephy6 — APPROVED** [review]

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/segment_edit/SegmentEdit.es.js (hunk 1 above)
Comment (claraizquierdo) [reply modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/segment_edit/SegmentEdit.es.js:553]: Hi! The PEDS Team said that adding aria-label and title always ensures more consistent reading so I think it is better to leave them both.

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Review by Mikellorza — APPROVED** [review]
Working! Now I can see the name of the toggle.

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Comment by claraizquierdo** [comment]
ci:stop

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Comment by rafaellajordao** [comment]
> Hi @CarlosBrichete and @rafaellajordao ,
> 
> could you have a look at the tests failing. They look related to the AC segments and not to the changes in this pull but better if you review it too. Thank you heart

Hey @claraizquierdo , these tests are not related with the changes :)

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Comment by claraizquierdo** [comment]
ci:forward

**Pull request closed without merging**